testapps/TestFunctionBuildProps/TestFunctionBuildProps/Readme.md

```diff
@@ -0,0 +1,49 @@
+# AWS Lambda Empty Function Project
```

Comment:
We don't really need the readme for checked in for the test apps.

Reply:
updated in latest commit